### PR TITLE
Test reliability

### PR DESCRIPTION
**Pull request by nategbrown9**
Turns off parallel execution and assigns each schedule timeline test a unique identifier to minimize likelihood of conflicts.

### CODE REVIEW

**Comment by nategbrown9** [comment]
Last 3 merges of main into this branch have resulted in successful test passes. Even if it's still not perfect, it seems to improve test reliability.

File: bridge-client/src/commonTest/kotlin/org/sagebionetworks/bridge/kmm/shared/repo/ScheduleTimelineRepoTest.kt [hunk 1]
```diff
@@ -666,7 +671,7 @@ class ScheduleTimelineRepoTest: BaseTest() {
     @Test
     fun testScheduledSessionsDay2WithCompletion() {
         runTest {
-            val studyId = "sage-assessment-test"
+            val studyId = "testScheduledSessionsDay2WithCompletiont"
```
Comment (Erin-Mounts) [inline bridge-client/src/commonTest/kotlin/org/sagebionetworks/bridge/kmm/shared/repo/ScheduleTimelineRepoTest.kt]: is the 't' at the end intentional and not a typo?

**Review by Erin-Mounts — APPROVED** [review]
One question/nit, otherwise looks good

**Review by syoung-smallwisdom — COMMENTED** [review]
So... tests are still failing?

**Pull request merged**